sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp

```diff
@@ -22,8 +25,33 @@ const vector<Tag> XMPFileFormat::ReadTags() {
   return XMPHelper::GetInstance().GetTags(mFile);
 }
 
-void XMPFileFormat::Commit(shared_ptr<IStream> file, string& /*newExtension*/) {
-  throw std::runtime_error("not implemented");
+void XMPFileFormat::Commit(shared_ptr<IStream> file, string& newExtension) {
```

Comment:
incorrect parameter names

Reply:
you mean shared_ptr<IStream> file? changed it to outputStream

---
In reply to: [139350563](https://github.com/AzureAD/rms-sdk-for-cpp/pull/156#discussion_r139350563) [](ancestors = 139350563)